Firestore/core/test/firebase/firestore/nanopb/byte_string_test.cc

```diff
@@ -39,7 +39,26 @@ std::vector<uint8_t> ToVector(const std::string& str) {
 
 TEST(ByteStringTest, DefaultConstructor) {
   ByteString str;
-  EXPECT_EQ(nullptr, str.data());
+  EXPECT_EQ(str.get(), nullptr);
+
+  // Even though the backing bytes array is null, data()/size() should be
```

Comment:
Nit: `size() should be non-null` probably needs to be rephrased.

Reply:
Removed `/size()`.